esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -205,6 +268,12 @@ void EsdlServiceImpl::init(const IPropertyTree *cfg,
 
         m_usesURLNameSpace = startsWith(m_serviceNameSpaceBase.str(), "http://") ? true : false;
 
+        const char* defaultFeatureAuth = srvcfg->queryProp("@defaultFeatureAuth");
+        if (isEmptyString(defaultFeatureAuth))
+            m_defaultFeatureAuth.append("${service}Access:full");
+        else if (!strieq(defaultFeatureAuth, "no_default"))
```

Comment:
This behavior looks good, but I want to confirm my understanding - the 'no_default' value would be a special value for the 'defaultFeatureAuth' attribute in the Service config element?